components/typography/Heading.js

```diff
@@ -46,7 +47,7 @@ export default function Heading({
     }
 
     return (
-        <Tag className={`${textColor} ${classNames}`}>
+        <Tag className={`${textColor} ${classNames}`} id={id}>
```

Comment:
why are adding id to this component?

Reply:
So that scrolling will work.

Reply:
For TOC to work, it's not necessary to add ids. We have implemented TOC in the docs too, kindly have a look into it and make the TOC accordingly. You can try to use the same component without making a separate component for the case studies.

Reply:
But the docs as well as blog is written in MarkDown, however case studies are written in yml. So there is no TOC component in the generated JSON. Hence the requirement for ids.